src/module/generate-swagger-spec.ts

```diff
@@ -1,6 +1,7 @@
 import * as ts from 'typescript';
 import * as YAML from 'yamljs';
 import { SwaggerConfig } from '../config';
+export { SwaggerConfig } from '../config';
```

Comment:
aren't we importing the same thing above?

Reply:
It looks similar but it's actually an export. I'm doing that so that users who are using tsoa programmatically are able to import the `SwaggerConfig` interface.

Otherwise users will have to resort to advanced types (`Parameters<T>`) like I did to be able to initialize a `SwaggerConfig` variable:
```ts
// I wish I could have written this as
// const swaggerGenArgs: SwaggerConfig = [
const swaggerGenArgs: Parameters<typeof tsoa.generateSwaggerSpec> = [
		{
			outputDirectory: staticFileServingDir,
			entryFile: './src/index.ts',
			host: 'localhost:3000',
			basePath: '/api/v1',
			yaml: true,
		},
	];

	const swaggerGenerationResult = await tsoa.generateSwaggerSpec(...swaggerGenArgs);
```